closure/repositories.bzl

```diff
@@ -227,33 +227,45 @@ def soyutils_usegoog():
       url = "https://bazel-mirror.storage.googleapis.com/repo1.maven.org/maven2/com/google/template/soy/2016-01-12/soy-2016-01-12-soyutils_usegoog.js",
   )
 
-def closure_repositories():
-  aopalliance()
+def closure_repositories_asm():
```

Comment:
I think the dependency groups should be reverted per the `omit_<dependency_name>` change.